spec.bs

```diff
@@ -1305,12 +1416,12 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
           1. Set |ig|'s [=interest group/ad components=] to |originalAdComponents|.
           1. If |generatedBid| is failure, [=iteration/continue=].
         1. [=list/Insert=] |generatedBid|'s [=generated bid/interest group=] in |bidIgs|.
+        1. Increment |bidsBeingScored| by 1.
         1. [=Score and rank a bid=] with |auctionConfig|, |generatedBid|, |leadingBidInfo|,
           |decisionLogicScript|, |dataVersion|, |auctionLevel|, |componentAuctionExpectedCurrency|, and |settings|'s
           [=environment/top-level origin=].
-
-  1. Decrement |pendingBuyers| by 1.
-1. Wait until |pendingBuyers| is 0.
+        1. Decrement |bidsBeingScored| by 1.
+1. Wait until |bidsBeingScored| is 0.

```

Comment:
I am not sure how this won't proceed while something didn't get to line 1419 yet?